x11-plugins/pidgin-telegram/pidgin-telegram-1.3.1.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="A libpurple protocol plugin that adds support for the Telegram messenger"
+HOMEPAGE="https://github.com/majn/telegram-purple"
+SRC_URI="https://github.com/majn/telegram-purple/releases/download/v${PV}/telegram-purple_${PV}.orig.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+IUSE="+gcrypt +icons +nls +webp +zlib"
+
+RDEPEND="net-im/pidgin
+	gcrypt? ( dev-libs/libgcrypt:0= )
+	!gcrypt? ( dev-libs/openssl:0= )
+	nls? ( sys-devel/gettext )
+	webp? ( media-libs/libwebp )
+	zlib? ( sys-libs/zlib )"
+
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/telegram-purple"
+
+src_prepare() {
+	# Remove '-Werror' to make it compile
```

Comment:
```
local WERROR_FILES=(
  Makefile.mingw ... # the whole list of files
)
local f
for f in "${WERROR_FILES[@]}"; do
    sed -i -e 's/-Werror //' "${f}" || die
done
```

Reply:
That are 3 lines more :P Anyway, I've changed it :)